prisma/schema.prisma

```diff
@@ -70,6 +70,14 @@ enum UserPlan {
   PRO
 }
 
+model UserDestinationCalendar {
+  id          Int    @id
+  integration String
+  externalId  String
+
+  user User @relation(fields: [id], references: [id])
+}
```

Comment:
Hey @KATT do you know what's the main difference between `UserDestinationCalendar` and `BookingDestinationCalendar` ?